# NGS
Some usefule Resources are list down based on their usage.

## Micro-Rna Analysis
1. http://bcb.io/2016/03/24/mirqc/

2. http://srna-workbench.cmp.uea.ac.uk/the-small-rna-workbench-version-4-3-1/

3. [DIANA Tools](http://snf-515788.vm.okeanos.grnet.gr/index.php?r=site/page&view=software)

  1. [MirPath 3](http://snf-515788.vm.okeanos.grnet.gr/)
  
 4. https://omictools.com/workflows-pipelines4-category

## Published Pipelines
1. systemPipeR: NGS workflow and report generation environment. [HTML] (https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1241-0), 
[Bioconductor](http://bioconductor.org/packages/devel/systemPipeR) 

2. https://peerj.com/articles/4466/?utm_source=summary_email_publication&utm_medium=email&utm_campaign=connection 
      https://github.com/oscar-franzen/rnaed.git
      
3. https://github.com/alesssia/YAMP

4. http://qiime.org/

5. https://f1000research.com/slides/6-1185

  https://www.nextflow.io/example4.html
6.  https://www.frontiersin.org/articles/10.3389/fcell.2014.00070/full


## Metagenomics

1. https://www.sciencedirect.com/science/article/pii/S0168165617315985

2. https://usegalaxy.org/u/aun1/p/windshield-splatter

3. FMAP: Functional Mapping and Analysis Pipeline for metagenomics and metatranscriptomics studies

4. http://clovr.org/methods/clovr-metagenomics/

5. https://arxiv.org/ftp/arxiv/papers/1604/1604.04103.pdf

6. http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0169563

7. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4426941/

8. https://github.com/CSB5/shotgun-metagenomics-pipeline

9. https://www.nature.com/articles/srep23774

10. https://github.com/jamesabbott/SCaMP
